### PR TITLE
Edgex 315 create a standalone edgex ekuiper snap

### PR DESCRIPTION
**Pull request by MonicaisHer**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -67,9 +67,16 @@ plugs:
...
-    content: edgex-secretstore-token
     target: $SNAP_DATA/mount/secrets
 
+slots:
+  kuiper-config:
+    interface: content
+    source:
+      read: 
+        - $SNAP/etc/sources
+        - $SNAP/etc/connections
```
Comment (farshidtz) [inline snap/snapcraft.yaml]: Change to:
```suggestion
        - $SNAP_DATA/kuiper/etc/sources
        - $SNAP_DATA/kuiper/etc/connections
```
Then connect
```
snap connect edgex-ekuiper:kuiper-config edgexfoundry:kuiper-config
```
```
$ sudo snap run --shell edgex-ekuiper.kuiper
# sudo ls /var/snap/edgex-ekuiper/current/etc/sources -l
# ls /var/snap/edgex-ekuiper/current/etc/sources -l
total 12
-rw-r--r-- 1 root root 8137 Feb 11 14:42 edgex.json
-rw-r--r-- 1 root root  440 Feb 11 15:37 edgex.yaml <-- note the modification time
```
Disconnect and compare the time:
```
# ls /var/snap/edgex-ekuiper/current/etc/sources -l
total 12
-rw-r--r-- 1 root root 8137 Feb 11 14:49 edgex.json
-rw-r--r-- 1 root root 1221 Feb 11 14:49 edgex.yaml
```

Comment (MonicaisHer) [reply snap/snapcraft.yaml]: Thanks. How to validate the content of modified `edgex.yaml` is correct inside the snap?

Comment (farshidtz) [reply snap/snapcraft.yaml]: You mean other than manually checking with cat or vi?

Comment (MonicaisHer) [reply snap/snapcraft.yaml]: Please ignore my comment. I was in the wrong place.

**Comment by MonicaisHer** [comment]
This PR will be ready after [edgex-ekuiper PR ](https://github.com/MonicaisHer/edgex-ekuiper-snap/pull/2)

**Pull request closed without merging**